Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -186,6 +186,18 @@
   categories:
   - UplinkAmmo
 
+# For the C20R
+- type: listing
+  id: UplinkMagazinePistolSubMachineGun
+  name: uplink-pistol-magazine-c20r-name
+  description: uplink-pistol-magazine-c20r-desc
+  icon: { sprite: /Textures/Objects/Weapons/Guns/Ammunition/Magazine/Pistol/smg_mag.rsi, state: red-icon }
+  productEntity: MagazinePistolSubMachineGun
+  cost:
+    Telecrystal: 3
```

Comment:
```suggestion
    Telecrystal: 5
```
3 TC for an SMG mag sounds overpowered to me, allowing a nuclear operative to get a total of 8 mags for the gun for 40 TC.

Reply:
But nukers can buy that amount of cartridges as it is, in pistol magazines. At this price there is no point in this magazine, I think they will just be patient and carry pistol magazines, which will give much more cartridges.

Reply:
How about at least 4?

Reply:
> But nukers can buy that amount of cartridges as it is, in pistol magazines.
That comes with the absolute awkwardness of moving cartridges from one mag to the other. You'd have little downtime as a nuclear operative to afford the time to shuffle cartridges from mag to mag.

Also, 3 pistol mags take more space than a single smg mag.
![image](https://github.com/space-wizards/space-station-14/assets/109166122/28890847-7cd9-4198-91cb-29803d2c3286)

Reply:
Yes, it is. But it doesn't seem that this way of balancing... strange? Forcing them to choke on uncomfortable magazines? Nukers are weak as it is, and at that price, I don't think anyone would take just a convenient magazine. Are you sure it's worth the price of 5?

Reply:
Just so it doesn't come out that no one will just buy it, because the price and quantity are absolutely unprofitable.